app/models/twilio_notifier.rb

```diff
@@ -13,7 +13,7 @@ def ordered_item_titles
   def notify
     client.messages.create(
       from: '+19495372518',
-      to: '+12026817334',
+      to: '+16082392471',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.